### PR TITLE
Update default bs variables

### PR DESCRIPTION
**Pull request by alovak**
@yazinsai take a look please.

the rule how to use it:

```
@import '/style-guide/base/variables';
@import '/bootstrap-sass-official/assets/stylesheets/bootstrap';
@import '/style-guide/main';
```

### CODE REVIEW

**Comment by yazinsai** [comment]
Why are we including BS variables in the style guide?

**Comment by alovak** [comment]
by setting BS variables we automatically has new style for the whole project without changing anything (almost anything).

After we set this variable ```$brand-primary``` we got correct hover state on button, panels, links, pagination, labels, etc.

**Comment by alovak** [comment]
I see why you asked this question. If the main purpose of this repo is to be used without bootsrap, then we can add separate file 'bootsrap' that will redefine bootsrap variables. So when we use bootstrap we can include it. What do you say?

**Comment by alovak** [comment]
@yazinsai I have updated files in PR. Take a look. New bootstrap.scss and one small change into old files.

**Comment by yazinsai** [comment]
LGTM

File: components/_button.scss [hunk 1]
```diff
@@ -58,3 +58,7 @@ $button-border-radius: $input-border-radius;
     background-color: start-color('primary', $opacity: 'light');
   }
 }
+
+.btn {
```
Comment (yazinsai) [inline components/_button.scss]: I think this is redundant for this PR

Usage today (before this PR): `.btn.start-button.secondary` 
Usage after this PR: `.btn.start-button.secondary`
Usage in the future (after some future PR which makes the `.start-button` class redundant): `.btn.secondary`

My suggestion would be to remove this block until the future PR is in.

Comment (alovak) [inline components/_button.scss]: In guide I just use ```.btn.btn-primary``` for this we need it.

Also after we merge this PR we can remove start-button from dashboard (to make button look good in different states).

Comment (yazinsai) [inline components/_button.scss]: CoolSent from my Huawei Mobile-------- Original Message --------Subject: Re: [style-guide] Update default bs variables (#2)From: Pavel Gabriel To: payfort/style-guide CC: Yazin In components/_button.scss:
> @@ -58,3 +58,7 @@ $button-border-radius: $input-border-radius;
>      background-color: start-color('primary', $opacity: 'light');
>    }
>  }
> +
> +.btn {

In guide I just use .btn.btn-primary for this we need it.

Also after we merge this PR we can remove start-button from dashboard (to make button look good in different states).

**Pull request merged**